### ISSUE TITLE
Searchbar is missing from UI description

### ISSUE THREAD

**Issue by odrobnik** [facebook/idb#892]
I have a `.searchable` bar on a nav bar.

<img width="2064" height="2752" alt="Image" src="https://github.com/user-attachments/assets/7f8dfd7f-a95b-4c23-89c7-d6fd4c1cc031" />

The UI description omits the Search Field embedded in the nav bar, as inspected with the Accessibility Inspector:

<img width="651" height="889" alt="Image" src="https://github.com/user-attachments/assets/ee24796c-054f-43f8-9fb9-7b0f10991ca7" />

In the accessibility output we can see that the nav bar doesn't have any children:

```
[
  {
    "AXFrame" : "{{0, 0}, {1032, 1376}}",
    "AXUniqueId" : null,
    "frame" : {
      "y" : 0,
      "x" : 0,
      "width" : 1032,
      "height" : 1376
    },
    "role_description" : "application",
    "AXLabel" : "SwiftLEGO",
    "content_required" : false,
    "type" : "Application",
    "title" : null,
    "help" : null,
    "custom_actions" : [

    ],
    "AXValue" : null,
    "enabled" : true,
    "role" : "AXApplication",
    "children" : [
      {
        "AXFrame" : "{{0, 32}, {1032, 166}}",
        "AXUniqueId" : "Andrea",
        "frame" : {
          "y" : 32,
          "x" : 0,
          "width" : 1032,
          "height" : 166
        },
        "role_description" : "Nav bar",
        "AXLabel" : null,
        "content_required" : false,
        "type" : "Group",
        "title" : null,
        "help" : null,
        "custom_actions" : [

        ],
        "AXValue" : null,
        "enabled" : true,
        "role" : "AXGroup",
        "children" : [

        ],
        "subrole" : null,
        "pid" : 57546
      },
```

There's something special about where in the hierarchy the Search Field is positioned. If I select it in Accessibility Inspector:

<img width="635" height="202" alt="Image" src="https://github.com/user-attachments/assets/4923616c-e2d8-4771-b451-e55ac171126b" />

Yet, if I select the Navbar itself, it doesn't have any children, which is probably why the search field is being skipped.

<img width="634" height="514" alt="Image" src="https://github.com/user-attachments/assets/de87b899-97eb-4a6a-ba01-84f847397a72" />
